sdcm/microbenchmarking.py

```diff
@@ -409,6 +415,8 @@ def parse_args():
                        help="Comma separated email addresses list that will get the report")
     check.add_argument("--report-path", action="store", default="",
                        help="Save HTML generated results report to the file path before sending by email")
+    check.add_argument("--build-url", action="store", default="", help='Set link to Jenkins build URL')
+    check.add_argument("--job-url", action="store", default="", help='Set link to Jenkins job URL')
```

Comment:
If we always get value from environment values, then we don't need to add those two arguments.

Reply:
Agree with Amos. Please remove those.

Reply:
Think about flexibility, if there is no Env variables. Ok. remove